resources/public/api/conf/1.0/headers.raml

```diff
@@ -18,3 +18,31 @@ traits:
         type: string
         required: true
         example: 'application/vnd.hmrc.1.0+json'
+  acceptJson:
```

Comment:
Can you remove this header as I believe we add this header

Reply:
removed